From 562df77cc7f025707278e2d0cdc838fa223cdbb5 Mon Sep 17 00:00:00 2001
From: Jeremie GARCIA <jeremiex.garcia@intel.com>
Date: Thu, 3 Jan 2013 11:42:38 +0100
Subject: [PORT FROM MAIN] wifi: enable CONFIG_MAC80211_DEBUGFS support

BZ: 74698

This patch enables suuport of CONFIG_MAC80211_DEBUGFS.
This will allow dynamic control of WMM-PS.

Change-Id: Iaefb8367865db1bcb459b4d3c7546c98ae6ec8c8
Orig-Change-Id: I97ab191e85fc6a7ead596e18b3ec21716153ec36
Signed-off-by: Jeremie GARCIA <jeremiex.garcia@intel.com>
Reviewed-on: http://android.intel.com:8080/84678
Reviewed-by: De Cesco, Jonathan <jonathan.de.cesco@intel.com>
Reviewed-by: Casasnovas, QuentinX <quentinx.casasnovas@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 wl12xx-compat/config.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/wl12xx-compat/config.mk b/wl12xx-compat/config.mk
index 3a07a68..7174d5d 100644
--- a/wl12xx-compat/config.mk
+++ b/wl12xx-compat/config.mk
@@ -166,7 +166,7 @@ ifndef CONFIG_COMPAT_KERNEL_2_6_33
 CONFIG_MAC80211_DRIVER_API_TRACER=y
 endif #CONFIG_COMPAT_KERNEL_2_6_33
 
-# CONFIG_MAC80211_DEBUGFS=y
+CONFIG_MAC80211_DEBUGFS=y
 # CONFIG_MAC80211_NOINLINE=y
 # CONFIG_MAC80211_VERBOSE_DEBUG=y
 # CONFIG_MAC80211_HT_DEBUG=y
-- 
1.7.4.1

